lago/virt.py

```diff
@@ -780,11 +782,14 @@ def copy_to(self, local_path, remote_path):
 
     def copy_from(self, remote_path, local_path, recursive=True):
         with self._scp() as scp:
-            scp.get(
-                recursive=recursive,
-                remote_path=remote_path,
-                local_path=local_path,
-            )
+            try:
+                scp.get(
+                    recursive=recursive,
+                    remote_path=remote_path,
+                    local_path=local_path,
+                )
+            except SCPException:
+                LOGGER.debug('Remote path %s not found' % remote_path)
```

Comment:
I wasn't sure if it should. I'm not sure an empty repo is something we can not continue with - the reason is that those RPMs might be elsewhere - in a different repo - so I opted to continue instead.